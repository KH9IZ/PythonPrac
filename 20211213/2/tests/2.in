[43, 254, 152, 4, 11, 207, 11, 17, 248, 54, 3, 115, 188, 93, 229, 45, 238, 229, 204, 22, 150, 284, 79, 139, 219, 65, 96, 39, 208, 33, 269, 287, 102, 104, 3, 8, 281, 231, 84, 154, 3, 34, 143, 203, 47, 287, 117, 273, 280, 156, 150, 245, 276, 82, 178, 85, 187, 298, 59, 184, 192, 34, 59, 275, 133]