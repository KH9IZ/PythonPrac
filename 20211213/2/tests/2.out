[3, 3, 3, 4, 8, 11, 11, 17, 22, 33, 34, 34, 39, 43, 45, 47, 54, 59, 59, 65, 79, 82, 84, 85, 93, 96, 102, 104, 115, 117, 133, 139, 143, 150, 150, 152, 154, 156, 178, 184, 187, 188, 192, 203, 204, 207, 208, 219, 229, 229, 231, 238, 245, 248, 254, 269, 273, 275, 276, 280, 281, 284, 287, 287, 298]